modules/openapi-generator/src/test/resources/bugs/issue_11957.yaml

```diff
@@ -0,0 +1,36 @@
+openapi: '3.0.2'
+info:
+  title: Search API
+  version: '1.0'
+servers:
+- url: http://localhost:8080/api/v1
+tags:
+- name: "search"
+paths:
+  /search:
+    get:
+      tags:
+      - search
+      description: ""
+      operationId: "search"
+      parameters:
+      - in: query
+        name: p
+        description: Page
+        schema:
+          type: integer
+          default: 0
+          minimum: 0
+      - in: query
+        name: orderBy
+        description: Filter by status not equal (AND condition applied)
+        schema:
+          type: array
+          default: ["updatedAt:DESC", "createdAt:DESC"]
```

Comment:
Could you add a test for an empty default as well?

```
default: []
```

Reply:
done